platform/admin/modules/admin-metrics-ui/pom.xml

```diff
@@ -166,7 +166,7 @@
         <dependency>
             <groupId>org.webjars.bower</groupId>
             <artifactId>moment</artifactId>
-            <version>2.10.3</version>
+            <version>2.18.1</version>
```

Comment:
This change will be rendered OBE by https://github.com/codice/ddf/pull/3030/files#diff-7ff44f74e6d015e0d183af70ccd2835fL169

Reply:
That's good.